presto-native-execution/presto_cpp/main/types/TypeSignatureTypeConverter.cpp

```diff
@@ -156,6 +156,10 @@ TypePtr typeFromString(const std::string& typeName) {
     upper = "DOUBLE";
   }
 
+  if (upper == "INTERVAL DAY TO SECOND") {
+    return INTERVAL_DAY_TIME();
+  }
+
```

Comment:
@kewang1024 
I see on the lines 141, 145 and 149 that we prefer using the TYPE()->toString() instead of hard-coding the string in one more place.
Let's use INTERVAL_DAY_TIME()->toString() here.